test_wazuh/test_fim/test_restrict/test_restrict_valid.py

```diff
@@ -71,15 +74,27 @@ def test_restrict(folder, filename, mode, content, triggers_event, tags_to_apply
     This test is intended to be used with valid configurations files. Each execution of this test will configure the
     environment properly, restart the service and wait for the initial scan.
 
-    :param folder string Directory where the file is being created
-    :param filename string Name of the file to be created
-    :param mode string same as mode in open built-in function
-    :param content string, bytes Content to fill the new file
-    :param triggers_event bool True if an event must be generated, False otherwise
-    :param tags_to_apply set Run test if match with a configuration identifier, skip otherwise
+    Parameters
+    ----------
+    folder : str
+        Directory where the file is being created
+    filename : str
+        Name of the file to be created
+    mode : str
+        Same as mode in open built-in function
+    content : str or bytes
+        Content to fill the new file
+    triggers_event : bool
+        True if an event must be generated, False otherwise
+    tags_to_apply : set
+        Run test if match with a configuration identifier, skip otherwise
     """
     check_apply_test(tags_to_apply, get_configuration['tags'])
 
+    if os.sep in filename:
```

Comment:
This approach is not correct since you are putting the folder inside the `filename` argument.